docs/connectors/sqlserver.asciidoc

```diff
@@ -433,7 +433,7 @@ The following _advanced_ configuration properties have good defaults that will w
 
 |`snapshot.locking.mode`
 |_none_
-|Controls how long the connector locks the monitored tables for snapshot execution. The default is _none_ which means that the connector does not hold any locks for all monitored tables. Using a value of _exclusive_ ensures that the connector holds the exclusive lock (and thus prevents any reads and updates) for all monitored tables.
+|Controls how long the connector locks the monitored tables for snapshot execution. The default is _none_ which means that the connector does not hold any locks for all monitored tables. Using a value of _exclusive_ ensures that the connector holds the exclusive lock (and thus prevents any reads and updates) for all monitored tables. _snapshot_ mode, in turn, runs the entire snapshot in snapshot transaction isolation level, thus neither table locks nor row-level locks are acquired. Limitation: DDL statements should not be executed during initial load in _snapshot_ mode.
```

Comment:
Can we say:

* "(and thus prevents any concurrent reads and updates)"
* "Concurrent DDL statements affecting captured tables must not be executed during initial load in _snapshot_ mode to avoid potentially inconsistent schema metadata"

Reply:
done